dll/win32/imm32/imm.c

```diff
@@ -3160,11 +3160,27 @@ BOOL WINAPI ImmEnumInputContext(DWORD idThread, IMCENUMPROC lpfn, LPARAM lParam)
  *              ImmGetHotKey(IMM32.@)
  */
 
+#ifdef __REACTOS__
+BOOL APIENTRY NtUserGetImeHotKey(DWORD, LPUINT, LPUINT, LPHKL); /* FIXME */
```

Comment:
`win32ss/include/ntuser.h` has this (but a worse prototype)

Reply:
OK.